src/types/quizzes/interfaces/quizzes.interface.ts

```diff
@@ -0,0 +1,22 @@
+import { RequestParams, CommonEntityFields } from '~/types'
+
+export interface QuizzesParams extends RequestParams {
+  title: string
+}
+
+export interface Answer {
+  text: string
+  isCorrect: boolean
+}
+
+export interface QuestionWithAnswers {
+  question: string
+  answers: Answer[]
+}
+
+export interface QuizInterface extends CommonEntityFields {
+  _id: string
```

Comment:
"_id" already exists in CommonEntityFields

Reply:
Fixed